acceptance/lib/puppet/acceptance/install_utils.rb

```diff
@@ -8,11 +8,12 @@ module Acceptance
     module InstallUtils
       PLATFORM_PATTERNS = {
         :redhat        => /fedora|el|centos/,
-        :debian        => /debian|ubuntu/,
+        :debian        => /debian|ubuntu|cumulus/,
         :debian_ruby18 => /debian|ubuntu-lucid|ubuntu-precise/,
         :solaris_10    => /solaris-10/,
         :solaris_11    => /solaris-11/,
         :windows       => /windows/,
+        :eos           => /eos/,
```

Comment:
Paranoid thought: make the regex match more restrictive, e.g. `/eos-4-i386/` or '/^eos-/' or something.

We're right now being bitten by the `el` match on line 10 inadvertently matching `elcapitan`, which has me thinking we should err a bit on the side of restricting matches. E.g., here, what if we start testing against `mirageos` in the future?

Thoughts? /cc @joshcooper @johnduarte